tinman/txgen.py

```diff
@@ -307,15 +322,35 @@ def port_snapshot(conf, keydb, silent=True):
 
 def build_actions(conf, silent=True):
     keydb = prockey.ProceduralKeyDatabase()
-
-    start_time = datetime.datetime.strptime(conf["start_time"], "%Y-%m-%dT%H:%M:%S")
+    stats_start = datetime.datetime.utcnow()
+    stats = account_stats(conf, silent)
```

Comment:
Name is too generic (`account_stats`) might be a good name.

Reply:
√